docker/docker_transport_test.go

```diff
@@ -44,9 +44,9 @@ func TestParseReference(t *testing.T) {
 func testParseReference(t *testing.T, fn func(string) (types.ImageReference, error)) {
 	for _, c := range []struct{ input, expected string }{
 		{"busybox", ""}, // Missing // prefix
-		{"//busybox:notlatest", "docker.io/library/busybox:notlatest"},           // Explicit tag
-		{"//busybox" + sha256digest, "docker.io/library/busybox" + sha256digest}, // Explicit digest
-		{"//busybox", "docker.io/library/busybox:latest"},                        // Default tag
+		{"//busybox:notlatest", "<local>/library/busybox:notlatest"},           // Explicit tag
+		{"//busybox" + sha256digest, "<local>/library/busybox" + sha256digest}, // Explicit digest
+		{"//busybox", "<local>/library/busybox:latest"},                        // Default tag
```

Comment:
This still essentially breaks the world; `skopeo inspect docker://busybox` will no longer do anything useful.